tests/performance/conftest.py

```diff
@@ -0,0 +1,9 @@
+import logging
+
+# pylint: disable-next=unused-wildcard-import,wildcard-import
```

Comment:
why?

Reply:
we need to import multiple fixtures from integration tests & mixins, explicitly listing them out is very verbose